tensorrt_llm/bench/benchmark/throughput.py

```diff
@@ -280,7 +321,7 @@ def throughput_command(
     # Runtime kwargs and option tracking.
     kwargs = {}
 
-    # Initialize the HF tokenizer for the specified model.
+    # Initialize the HF tokenizer for the specified model. This is only used for data preparation.
     tokenizer = initialize_tokenizer(checkpoint_path)
```

Comment:
We have a `no_skip_tokenizer_init` flag above, but this is called unconditionally as far as I can see. Is that expected?